downloads.md

```diff
@@ -68,6 +68,10 @@ Once you've downloaded Spark, you can find instructions for installing and build
 
 <ul id="sparkReleaseNotes"></ul>
 
+### Archived Releases
+
+As new Spark releases comes out for each development stream, previous ones will be archived but they are still available at [Spark release archives](http://archive.apache.org/dist/spark/).
```

Comment:
releases comes -> releases come
archived but -> archived, but
Also for good measure, make the link an https URL? It works and I suppose we should point people to a secure URL for obtaining software.